internal/rpcserver/interface.go

```diff
@@ -410,6 +410,15 @@ type Chain interface {
 	// TipGeneration returns the entire generation of blocks stemming from the
 	// parent of the current tip.
 	TipGeneration() ([]chainhash.Hash, error)
+
+	// IsTreasuryAgendaActive wraps blockchain.IsTreasuryAgendaActive.
+	IsTreasuryAgendaActive() (bool, error)
```

Comment:
We don't always get a hash in the incoming command. I am open to remove it with the explicit expectation that this will require two commands to complete. Get tip hash -> check if treasury is enabled.

That ok?

Reply:
Removed interface function and force use of `IsTreasuryAgendaActiveByHash`